endpoints/web.py

```diff
@@ -690,6 +690,7 @@ def request_authorization_code():
     redirect_uri = request.args.get("redirect_uri", None)
     scope = request.args.get("scope", None)
     state = request.args.get("state", None)
+    code = request.args.get("code", None)
```

Comment:
The code shouldn't be included as part of the authorize request step of oauth. It's what we the Quay app would return as part of the uri when redirecting. The `code` is already be included, based on `oauth/provider.py`